jdaviz/core/template_mixin.py

```diff
@@ -608,3 +644,185 @@ class ViewerSelectMixin(VuetifyTemplate, HubListener):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.viewer = ViewerSelect(self, 'viewer_items', 'viewer_selected')
+
+
+class DatasetSelect(BaseSelectPluginComponent):
+    """
+    Traitlets (in the object, custom traitlets in the plugin):
+
+    * ``items`` (list of dicts with keys: )
```

Comment:
Is there supposed to be something after the colon here?

Reply:
good catch, thanks! (added in the latest commit)